solution/data/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
Ah, ok, I forgot about that. Is `data/` still used in any of the labs? I vaguely remember this from earlier versions of course, but wasn't sure if it's still required.

Reply:
We had a chat about sqlite getting replaced with Postgres. I looked in all of the docker-compose files and we no longer mount the `data` folder anywhere. Looks like it can go.